src/console.cpp

```diff
@@ -117,57 +117,89 @@ void IConsolePrint(TextColour colour_code, const char *string)
 	IConsoleGUIPrint(colour_code, str);
 }
 
+void CDECL IConsolePrintV(TextColour colour_code, const char *format, va_list args)
+{
+	assert(IsValidConsoleColour(colour_code));
+
+	char buf[ICON_MAX_STREAMSIZE];
+	vseprintf(buf, lastof(buf), format, args);
```

Comment:
Need to come up with a solution to this warning... which may well be just disabling this warning at this point. This must be the case somewhere else, see if you can find anything?

Reply:
I think I've fixed the compile issue using the "magic" WARN_FORMAT macro. It's a bit hard to test locally as I only am developing on Windows.